PINCE.py

```diff
@@ -1106,6 +1106,10 @@ def scan_values(self):
         if debugcore.currentpid == -1:
             return
         search_for = self.validate_search(self.lineEdit_Scan.text(), self.lineEdit_Scan2.text())
+        
+        if self.comboBox_ScanType.currentData(Qt.ItemDataRole.UserRole) == typedefs.SCAN_TYPE.NOT:
```

Comment:
This logic should be in `validate_search`. It looks ugly being the only one outside, plus if validate_search is ever used elsewhere, the Not functionality will be broken.

Reply:
I see, will try to put it there.